csharp/ParquetSharp.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6;net471;netstandard2.1</TargetFrameworks>
-    <LangVersion>9.0</LangVersion>
+    <TargetFrameworks>net8.0;net10.0;net471;netstandard2.1</TargetFrameworks>
```

Comment:
I think dropping net6 makes sense as this is now out of support, but is there any reason to build for both net8 and net10? I would have thought we should just build for net8 and net10 users can use the net8 build. I guess the increase in the size of the nuget package is minimal so there aren't really any downsides to this though.